### PR TITLE
Support for flow stats in 0.7.x

### PR DESCRIPTION
**Pull request by ANISH-GOTTAPU**
Support for flow stats in 0.7.x

### CODE REVIEW

File: snappi_ixnetwork/trafficitem.py [hunk 1]
```diff
@@ -1277,48 +1293,97 @@ def results(self, request):
...
-                flow_row = flow_rows[name]
+        if self._api._flow_tracking:
+            table = self._api.assistant.StatViewAssistant("Flow Statistics")
+            for row in table.Rows:
+                name = row["Traffic Item"]
                 if (
-                    float(row["Tx Frame Rate"]) > 0
-                    or int(row["Tx Frames"]) == 0
+                    len(flow_names) > 0
+                    and row["Traffic Item"] not in flow_names
```
Comment (Rangababu-R) [inline snappi_ixnetwork/trafficitem.py:1302]: Why is the logical statement made to "AND" logic instead of "OR"?

**Review by Rangababu-R — CHANGES_REQUESTED** [review]

File: snappi_ixnetwork/trafficitem.py (hunk 1 above)
Comment (ANISH-GOTTAPU) [reply snappi_ixnetwork/trafficitem.py:1302]: even though flow_names>0 condition satisfies, the `flow_name` requet by the user should be part of flow_names

**Review by Rangababu-R — APPROVED** [review]

File: do.py [hunk 2]
```diff
@@ -53,8 +53,8 @@ def test():
         (
             '--ports="otg-novus100g.lbj.is.keysight.com;1;1'
             " otg-novus100g.lbj.is.keysight.com;1;2"
-            " otg-novus100g.lbj.is.keysight.com;1;3"
-            ' otg-novus100g.lbj.is.keysight.com;1;4"'
+            " otg-novus100g.lbj.is.keysight.com;1;5"
```
Comment (alakendu) [inline do.py:56]: please revert it when the port will available

Comment (ANISH-GOTTAPU) [reply do.py:56]: yeah..sure

**Pull request merged**